app/src/main/java/com/futurice/freesound/feature/search/SoundItemViewHolder.java

```diff
@@ -29,11 +29,11 @@
 import android.view.View;
 import android.widget.TextView;
 
-import rx.android.schedulers.AndroidSchedulers;
-import rx.subscriptions.CompositeSubscription;
+import io.reactivex.disposables.CompositeDisposable;
```

Comment:
Have you set the formatter for the project? I ran a reformat then and it moved some lines around. Use the formatter settings in the repo: https://github.com/futurice/freesound-android/tree/master/ide/android-studio/code-style